assignments.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Assignments and Projects</title>
+		<link href="css/main.css" rel="stylesheet" type="text/css"/>
+	</head>
+	<body>
+		<h1 class="centre">Assignments and Projects</h1>
+		
+		<article>	
+			<section>
+				<h2>Introduction</h2>
```

Comment:
It would be good to break down each section into individual pages, allowing for more detailed tip for fellow students